src/utils/constants.ts

```diff
@@ -0,0 +1,22 @@
+// Actions constants
+
+const GET_USER_CREDENTIALS: 'GET_USER_CREDENTIALS' = 'GET_USER_CREDENTIALS';
```

Comment:
если тайпскрипт без указания типа говорит, что константа типа string. то можно сделать так
```suggestion
const GET_USER_CREDENTIALS = 'GET_USER_CREDENTIALS' as const;
```